### PR TITLE
Runtime class name

### PR DESCRIPTION
**Pull request by Francesko90**
---
**Runtime class name**

Currently, there is no possibility to add runtime class name to worker pod, In my case is necessary to use NVIDIA runtime as class name to use NVIDIA driver and GPU for this reason I thought to add runtimeClassName to helm chart for worker deployment

### CODE REVIEW

**Review by phanikumv — COMMENTED** [review]
Can we add unit tests for this please ? Please check existing tests under`/airflow/tests/charts`

**Comment by jedcunningham** [comment]
Also don't forget to add this to `values.yaml` and `files/pod-template-file.kubernetes-helm-yaml`.

**Comment by Francesko90** [comment]
I updated `values.yaml` and `files/pod-template-file.kubernetes-helm-yaml.` 
I created unit test but I'm not sure that it is correct, let me know if there are any problem,

File: tests/charts/airflow_core/test_worker.py [hunk 1]
```diff
@@ -373,6 +373,17 @@ def test_should_create_default_affinity(self):
...
+    def test_runtime_class_name_values_are_configurable(self):
+        docs = render_chart(
+            values={
+                "workers": {"runtimeClassName": "nvidia"},
+            },
+            show_only=["templates/workers/worker-deployment.yaml"],
+        )
+
+        runtimeclassname = jmespath.search("spec.template.spec.runtimeClassName", docs[0])
+        assert runtimeclassname == "nvidia"
```
Comment (amoghrajesh) [inline tests/charts/airflow_core/test_worker.py]: You can just do an `assert jmespath.search("spec.template.spec.runtimeClassName", docs[0]) == "nvidia"` here

File: chart/values.yaml [hunk 2]
```diff
@@ -2056,36 +2062,36 @@ config:
...
-    reinit_frequency: '{{ .Values.kerberos.reinitFrequency }}'
-    principal: '{{ .Values.kerberos.principal }}'
-    ccache: '{{ .Values.kerberos.ccacheMountPath }}/{{ .Values.kerberos.ccacheFileName }}'
+    keytab: "{{ .Values.kerberos.keytabPath }}"
+    reinit_frequency: "{{ .Values.kerberos.reinitFrequency }}"
+    principal: "{{ .Values.kerberos.principal }}"
+    ccache: "{{ .Values.kerberos.ccacheMountPath }}/{{ .Values.kerberos.ccacheFileName }}"
   celery_kubernetes_executor:
-    kubernetes_queue: 'kubernetes'
+    kubernetes_queue: "kubernetes"
```
Comment (amoghrajesh) [inline chart/values.yaml]: Were all these changes from single quotes to double quotes done by mistake? Looks to me as if the editor did it.

File: chart/values.yaml [hunk 3]
```diff
@@ -1950,7 +1951,12 @@ cleanup:
   # Command to use when running the cleanup cronjob (templated).
   command: ~
   # Args to use when running the cleanup cronjob (templated).
-  args: ["bash", "-c", "exec airflow kubernetes cleanup-pods --namespace={{ .Release.Namespace }}"]
+  args:
+    [
+      "bash",
+      "-c",
+      "exec airflow kubernetes cleanup-pods --namespace={{ .Release.Namespace }}",
+    ]
```
Comment (amoghrajesh) [inline chart/values.yaml]: Same comment as below

File: chart/values.yaml (hunk 2 above)
Comment (Francesko90) [reply chart/values.yaml]: Sorry about that; I think that is the fault of Linter.

**Review by potiuk — APPROVED** [review]

**Pull request merged**